README.md

```diff
@@ -33,27 +33,27 @@ To use the [plugin](https://plugins.gradle.org/plugin/co.com.bancolombia.cleanAr
 
 ```groovy
 plugins {
-    id "co.com.bancolombia.cleanArchitecture" version "2.4.9"
+    id "co.com.bancolombia.cleanArchitecture" version "2.4.10"
```

Comment:
Hi @gabheadz this version updates are not required, because the release action will update it automatically.

Please revert all changes related to versión.

Reply:
Done. Manual versioning reverted with commit [fix(version): undo manual version id setting](https://github.com/bancolombia/scaffold-clean-architecture/pull/309/commits/5809a9c81e4426e2445e8aa2c3efb4b1ed107edf)